### ISSUE TITLE
WhereInIds no longer works in the latest version.

### ISSUE THREAD

**Issue by stonecold123** [typeorm/typeorm#594]
I just updated to the latest 0.1.0-alpha.23 from `0.09`. I found that `whereInIds` no longer  returns any rows like it used to.

The following example used to work with `createQueryBuilder()`:

````
    getMultipleItems(mids: number[]): Promise<Item[]> {
            let query =  this.ormRepository
                .createQueryBuilder("item")
                .whereInIds(mids)
                //.orderBy(`FIELD(item.mid, ${mids.join(',')})`)
            let q = query.getQuery();
            return query.getMany()
    }
````
It doesn't return any rows. I need to use querybuilder for a single table query in order to use orderBy clause and groupBy clause in my situation. 

The raw query looks fine. I can copy the query and retrieve rows from a database gui client like DBeaver. Maybe there is some placeholder matching problem?

````
SELECT `item`.`mid` AS `item_mid`,`item`.`name` AS `item_name` FROM item_info item
WHERE (`item`.`mid`=:id_0_0 OR `item`.`mid`=:id_1_0 OR `item`.`mid`=:id_2_0 OR `item`.`mid`=:id_3_0 OR `item`.`mid`=:id_4_0 OR `item`.`mid`=:id_5_0 OR `item`.`mid`=:id_6_0 OR `item`.`mid`=:id_7_0 OR `item`.`mid`=:id_8_0 OR `item`.`mid`=:id_9_0 OR `item`.`mid`=:id_10_0 OR `item`.`mid`=:id_11_0 OR `item`.`mid`=:id_12_0 OR `item`.`mid`=:id_13_0 OR `item`.`mid`=:id_14_0 OR `item`.`mid`=:id_15_0 OR `item`.`mid`=:id_16_0 OR `item`.`mid`=:id_17_0 OR `item`.`mid`=:id_18_0 OR `item`.`mid`=:id_19_0) 
````

This one is able to retrieve results:
````
getMultipleItems(mids: number[]): Promise<Item[]> {
       return this.ormRepository.findByIds(mids);
}
````

**Comment by pleerock** [comment]
can you please check if parameters are correct? query looks correct isn't it? you can use `query.getSql();` method to get sql with parameters replaced

**Comment by stonecold123** [comment]
@pleerock 
When using `query.getSql()`, it returns this query:

SELECT `item`.`mid` AS `item_mid`,`item`.`name` AS `item_name` FROM `item_info` `item` WHERE (`item`.`mid`=? OR `item`.`mid`=? OR `item`.`mid`=? OR `item`.`mid`=? OR `item`.`mid`=? OR `item`.`mid`=? OR `item`.`mid`=? OR `item`.`mid`=? OR `item`.`mid`=? OR `item`.`mid`=? OR `item`.`mid`=? OR `item`.`mid`=? OR `item`.`mid`=? OR `item`.`mid`=? OR `item`.`mid`=? OR `item`.`mid`=? OR `item`.`mid`=? OR `item`.`mid`=? OR `item`.`mid`=? OR `item`.`mid`=?) 

No error, but no results returned. 
I have to rewrite it to this to make it work:

Example:
````
// mids = [333,123,456];
let query =  this.ormRepository.createQueryBuilder("item");
mids.forEach((mid,idx)=>{
      query.orWhere(`mid = :m${idx}`,{['m'+idx]:mid})
});
````

**Comment by pleerock** [comment]
Can you provide a complete small example how to reproduce this problem?

**Comment by stonecold123** [comment]
Please check this example:
````
import {Column, JoinColumn, OneToOne, PrimaryGeneratedColumn} from "typeorm";
import {Entity} from "typeorm/decorator/entity/Entity";

@Entity()
export class ModelReview  {
    @PrimaryGeneratedColumn()
    postId: number;

    @Column({type:'int'})
    modelId:number;
}

````
````
import {Repository} from "typeorm";
import {Service} from "typedi";
import {OrmRepository} from "typeorm-typedi-extensions";
import {ModelReview} from "../entity/Sample";

@Service()
export class SampleRepository {

    constructor(@OrmRepository(ModelReview) private ormRepository: Repository<ModelReview>) {}

    public async getItem():Promise<ModelReview[]>{
        let postId = [1,2,5];
        let result = await this.ormRepository.createQueryBuilder("r")
            .whereInIds(postId)
            .getMany()
        return result;
    }
}
````

**Comment by pleerock** [comment]
Fix for this issue is released in `0.1.0-alpha.26`
